src/components/questionary/questionaryComponents/QuestionnairesList.tsx

```diff
@@ -47,42 +60,50 @@ export function QuestionnairesList({
   addButtonLabel,
 }: QuestionnairesListProps) {
   const classes = useStyles();
+  const isListEmpty = data.length === 0;
 
   return (
     <div>
-      <List
-        component="ul"
-        className={classes.questionnairesList}
-        style={{ ...style }}
-      >
-        {data.map((record) => {
-          return (
-            <QuestionnairesListItem
-              record={record}
-              onEditClick={onEditClick}
-              onDeleteClick={onDeleteClick}
-              onCloneClick={onCloneClick}
-              key={record.id}
-            />
-          );
-        })}
-      </List>
-      <ButtonContainer>
-        <Button
-          onClick={onAddNewClick}
-          variant="outlined"
-          data-cy="add-button"
-          size="small"
-          color="primary"
-          startIcon={<AddCircleOutlineIcon />}
-          disabled={
-            (!!maxEntries && data.length >= maxEntries) ||
-            onAddNewClick === undefined
-          }
+      {isListEmpty ? (
+        <div className={classes.emptyList}>The list is empty</div>
+      ) : (
+        <List
+          component="ul"
+          className={classes.questionnairesList}
+          style={{ ...style }}
         >
-          {addButtonLabel || 'Add'}
-        </Button>
-      </ButtonContainer>
+          {data.map((record) => {
+            return (
+              <QuestionnairesListItem
+                record={record}
+                onEditClick={onEditClick}
+                onDeleteClick={onDeleteClick}
+                onCloneClick={onCloneClick}
+                key={record.id}
+              />
+            );
+          })}
+        </List>
+      )}
+      <div className={classes.bottomBar}>
```

Comment:
The same with the Box component goes here it is easier to understand it it is written like: `<Box display="flex" alignItems="center">`